src/main/java/net/minecraftforge/fml/common/registry/VillagerRegistry.java

```diff
@@ -250,18 +181,17 @@ private void init()
         }
     }
 
-    public static class VillagerProfession
+    public static class VillagerProfession extends IForgeRegistryEntry.Impl<VillagerProfession>
```

Comment:
Since `VillagerProfession` and `VillagerCareer` are now useful in their own right, would it be too far fetched to see them moved into their own file instead of remaining as nested classes?